src/vm/amd64/asmhelpers.S

```diff
@@ -52,16 +52,9 @@
 //   Register preservation scheme:
 //
 //       Preserved:
-//           - all non-volatile registers
-//           - rax, rdx
-//           - xmm0, xmm1
-//
-//       Not Preserved:
-//           - integer argument registers (rcx, rdx, r8, r9)
-//           - floating point argument registers (xmm1-3)
-//           - volatile integer registers (r10, r11)
-//           - volatile floating point registers (xmm4-5)
-//           - upper halves of ymm registers on AVX (which are volatile)
+//           - All argument and callee-saved registers must be preserved:
+//             RDI, RSI, RDX, RCX, R8, R9,  RBX, RBP, R12�R15, XMM0, XMM1, XMM2, XMM3, XMM4, XMM5, XMM6, XMM7
```

Comment:
The character between R12 and R15 shows up as garbage when I view this in CodeFlow; I wonder if it's some Unicode thing.